methods/get_seqrun_meta.nf

```diff
@@ -1,9 +1,21 @@
 // Function for sequencing_run & clarity_sample_id
 def get_seqrun_meta(LinkedHashMap row) {
-    if (row.sequencing_run && row.clarity_sample_id) {
-        meta = tuple(row.id, row.sequencing_run, row.clarity_sample_id)
+    if (row.sequencing_run) {
+        meta = [row.id, row.sequencing_run]
+        if (row.clarity_sample_id) {
+            meta << row.clarity_sample_id
```

Comment:
What does `<<` do here? Isn't it a bitshift operator? (https://docs.groovy-lang.org/latest/html/documentation/core-operators.html#_bit_shift_operators)

Reply:
It's an append operator to a list in groovy